pkg/controller/servicemesh/controlplane/integration_test.go

```diff
@@ -75,7 +75,7 @@ func TestBootstrapping(t *testing.T) {
 		operatorNamespace     = "istio-operator"
 		controlPlaneNamespace = "test"
 		smcpName              = "test"
-		cniDaemonSetName      = "istio-node"
+		cniDaemonSetName      = "istio-cni-node"
```

Comment:
that needs docs update here: https://docs.openshift.com/container-platform/4.10/service_mesh/v2x/removing-ossm.html